frontend/storybook/src/components/entry/publish/publish-two.ts

```diff
@@ -0,0 +1,73 @@
+import "@elements/entry/jig/publish/page-2";
+import "@elements/entry/home/TOSORT/image-thumbnail"; 
+import "@elements/entry/home/TOSORT/placeholder"; 
+import "@elements/widgets/tags/icon";
+import "@elements/entry/jig/publish/publish-dropdown";
+import "@elements/core/inputs/textarea";
+import "@elements/core/dividers/spacer-fourty";
+import "@elements/core/titles/variants/title-w-icon";
+import { Rectangle } from "~/components/core/buttons/rectangle";
+import { colorStyles } from "@elements/_styles/colors";
+
+export default {
+  title: 'Full Pages/Publish',
+}
+
+  
+  interface PublishArgs {
+    closed:boolean,
+  
+    }
+
+    const DEFAULT_ARGS:PublishArgs = {
+       closed:false,
+      }
+
+      const STR_TITLE = "Settings and JIG info.";
+      const STR_SUBTITLE = "Last step before publishing";
+      const STR_MEDIUM = "medium";
+      const STR_WHITE = "white";
+      const STR_SHARE = "share the JIG";
+      const STR_CREATE = "create a new JIG";
+      const STR_PLAY = "play the JIG";
+      const STR_ICNSHARE = "Icn_Share_Red.svg";
+      const STR_ICNCREATE = "Icn_Plus_Red.svg";
+      const STR_ICNPLAY = "icn-video-activity-hover.svg";
+      const STR_DROPDOWNTITLE = "Select Share Option";
+      const STR_STUDENT = "icn-student.svg";
+      const STR_SHARESTUDENT ="Share with Students";
+      const STR_URLICON = "icn-url.svg";
+      const STR_URL ="Copy URL";
+
+      
+
+
+
+export const PublishFullTwo = (props?:PublishArgs) => {
+
+ const {} = props || DEFAULT_ARGS;
+ 
+
+    return `
+    <publish-page-two title="${STR_TITLE}" subtitle="${STR_SUBTITLE}">
```

Comment:
static content should be in the element not the component. applies here and other text elements below